command/debug.go

```diff
@@ -0,0 +1,686 @@
+package command
+
+import (
+	"archive/tar"
+	"compress/gzip"
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/hashicorp/nomad/api"
+	"github.com/posener/complete"
+)
+
+type DebugCommand struct {
+	Meta
+
+	timestamp   string
+	collectDir  string
+	duration    time.Duration
+	interval    time.Duration
+	logLevel    string
+	nodeIDs     []string
+	serverIDs   []string
+	consulToken string
+	vaultToken  string
+	manifest    []string
+	stopCh      chan struct{}
+}
+
+const (
+	userAgent = "nomad debug"
+)
+
+func (c *DebugCommand) Help() string {
+	helpText := `
+Usage: nomad debug [options]
+
+  Build an archive containing Nomad cluster configuration and state information, Nomad
+  server and task logs, and logs from the host systems running Nomad servers and clients. If
+  no selection option is provided, the debug archive contains logs from all nodes in the
```

Comment:
Haven't read the implementation yet, but if `debug logs from all nodes in the cluster` is accurate, that seems like a dangerous default. Debug logs on clients can be very noisy.

Reply:
nope, that's stale docs from the earlier RFC!

Reply:
Fixed this doc string, I think we'll go with the smaller bundle for now and we can see if we have a routine problem that would be fixed by including more logs.